### PR TITLE
Pump/Fractionating tower fixes

### PR DESCRIPTION
**Pull request by Anikyte**
Fixed some problematic behaviour with pumps, and fixed the fractionating tower whilst adding a couple small improvements.

### CODE REVIEW

File: src/main/java/net/snakefangox/mechanized/blocks/entity/SteamFractionatingTowerEntity.java [hunk 1]
```diff
@@ -80,31 +78,15 @@ private void doMine() {
 			world.breakBlock(minePos, false);
 			List<ItemEntity> items = world.getEntities(ItemEntity.class, new Box(minePos), null);
 			items.forEach(ie -> {
-				double x = ie.getX();
-				double z = ie.getZ();
+				double x = (ie.getX() - pos.getX()) / 2 + pos.getX();
+				double z = (ie.getZ() - pos.getZ()) / 2 + pos.getZ();
```
Comment (Anikyte) [inline src/main/java/net/snakefangox/mechanized/blocks/entity/SteamFractionatingTowerEntity.java:82]: This effectively shrinks the area items are teleported to.

**Comment by snakefangox** [comment]
I have a bit of a dilemma as to how to handle this. Pressure needing to be greater than the cost of operation for some machines is an intended game mechanic, however you're right that it's not obvious how that works and I'm not sure it improves the mod overall. I've been planning to do something proper with pressure since forever but as might be clear I've had near zero time for working on this mod recently. 

As far as the other changes you made you're spot on, people have been complaining about the valid check since forever and operating at longer intervals for those machines makes sense. If you don't mind I plan on merging and releasing this and then working on a 1.16.3 version with a few changes, possible altering how some of this stuff works.  All of which has been a very longwinded way of saying "Looks good to me".

**Comment by Anikyte** [comment]
I figured it might have been intended, but the pump in specific consumed so little steel, and yet required so much pressure that it ended up causing a lot of problems automating in the early game. I live the idea that machines need pressure to work, but the pump specifically shouldn't. I think automation mods should be able to be automated early on, and a pump is a key component of that. And anyway, realistically, a pump functioning off pressure differential would likely build up steam over time and release it to create a temporary vacuum. Or maybe a sterling engine sort of
deal.

**Comment by snakefangox** [comment]
Ah yeah, I can see the pressure requirement being a pain early game. I think you're right that the pump shouldn't have a pressure limit.

**Pull request merged**